src/include/units/one_rep.h

```diff
@@ -82,7 +82,10 @@ struct one_rep {
   template<QuantityValue Rep>
   [[nodiscard]] friend constexpr Rep operator/(one_rep, const Rep&) = delete;
 
-  template<QuantityValue Rep>
+  template<typename Rep>
+    requires requires { quantity_values<Rep>::one(); requires !Quantity<Rep> && // TODO: Replace with `QuantityValue`
+                                                              !QuantityLike<Rep> && // when Clang catches up.
+                                                              !wrapped_quantity_<Rep>; }
```

Comment:
Maybe the postfix contraints should be put outside of the first requires expressions (just leave checking for `one` there)?